code/datums/traits/negative.dm

```diff
@@ -59,7 +59,7 @@
 	medical_record_text = "Patient has a tumor in their brain that is slowly driving them to brain death."
 
 /datum/quirk/brainproblems/on_process()
-	quirk_holder.adjustOrganLoss(ORGAN_SLOT_BRAIN, 0.2)
+	quirk_holder.adjustOrganLoss(ORGAN_SLOT_BRAIN, STANDARD_ORGAN_THRESHOLD*0.2)
```

Comment:
hey, if you're going to nerf a perk to oblivion, maybe you should mention it anywhere. constant unstoppable brain damage reaching a cap versus killing you is a big deal.

Reply:
im stupid and thought i was fixing a bug lol let me revert that

Reply:
i was looking for the part where it added brain damage and was confused it wasn't there, so that makes sense (sorry for blowing up i get very annoyed about undocumented stuff)